packages/ui/react/src/components/Forms/Label/Label.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+
+import { Tooltip } from '../../Primitives'
```

Comment:
Can you apply the suggestion below to use path alias instead of relative path?

```suggestion
import { Tooltip } from 'src/components/Primitives'
```